2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs

```diff
@@ -85,11 +86,9 @@ public void ConfigureServices(IServiceCollection services)
                     };
                 }, options =>
```

Comment:
Remove cookie action after PR [309](https://github.com/AzureAD/microsoft-identity-web/pull/309) is merged.